bundler.gemspec

```diff
@@ -32,4 +32,6 @@ Gem::Specification.new do |s|
   s.bindir        = "exe"
   s.executables   = %w(bundle bundler)
   s.require_paths = ["lib"]
+
+  s.post_install_message = "Bundler and RubyGems.org are free for anyone to use, but maintaining them costs more than $20,000 USD every month. If you make a profit using Ruby, help us cover those costs so that we can keep the gem ecosystem free for everyone: https://rubytogether.org/join"
```

Comment:
@indirect I totally understand where you're coming as I've followed things this year and knew what you were trying to do with that message. I agree that my example is not as striking as it could be but the tone of the current message can definitely rub people the wrong way and reflect poorly on Rubytogether.org. 
I've had a pretty bad day today, but if you're up for it, give me a couple days and see if I can help you come up with something better with a stronger appeal factor.